src/index.js

```diff
@@ -145,7 +150,7 @@ var ReactGA = {
     }
 
     if (typeof ga === 'function') {
-      ga('send', 'pageview', path);
+      ga((_trackerName == '' ? '' : _trackerName + '.') + 'send', 'pageview', path);
```

Comment:
Probably can extract this appending logic into a function since it's used in multiple places.